source/fundamentals/connection.txt

```diff
@@ -214,7 +235,15 @@ URI to specify the behavior of the client.
      - Specifies whether to establish a Transport Layer Security (TLS)
        connection with the instance. This is automatically set to ``true``
        when using a DNS seedlist (SRV) in the connection string. You can
-       override this behavor by setting the value to ``false``.
+       override this behavior by setting the value to ``false``.
+
+   * - **useUnifiedTopology** 
+     - boolean 
+     - ``true``
+     - Specifies whether to use the **unified topology** layer. To
+       learn what the unified topology layer is and why it was introduced,
+       see `this document <https://github.com/mongodb/node-mongodb-native/blob/3.6/docs/reference/content/reference/unified-topology/index.md>`__
+       produced by the MongoDB Node driver engineering team. 
```

Comment:
```suggestion
       . 
```

Reply:
Thanks for the note. Discussed this suggestion on slack, and I agree it doesn't add much for users to mention the engineering team. Fix: https://github.com/mongodb/docs-node/pull/149/commits/6601707dfc483757b82c83aa2ca4ce3e001450f6